BUILD.md

```diff
@@ -174,6 +157,9 @@ cuML's cmake has the following configurable flags available:
 | BLAS_LIBRARIES | path/to/blas_lib | "" | Optional variable allowing to manually specify location of BLAS library. |
 | BUILD_CUML_CPP_LIBRARY | [ON, OFF]  | ON  | Enable/disable building libcuml++ shared library. Setting this variable to `OFF` sets the variables BUILD_CUML_C_LIBRARY, BUILD_CUML_TESTS, BUILD_CUML_MG_TESTS and BUILD_CUML_EXAMPLES to `OFF` |
 | BUILD_CUML_C_LIBRARY | [ON, OFF]  | ON  | Enable/disable building libcuml shared library. Setting this variable to `ON` will set the variable BUILD_CUML_CPP_LIBRARY to `ON` |
+| BUILD_CUML_STD_COMMS | [ON, OFF] | ON | Enable/disable building cuML NCCL+UCX communicator for running multi-node multi-GPU algorithms. Note that UCX support can also be enabled/disabled (see below) |
+| WITH_UCX | [ON, OFF] | OFF | Enable/disable UCX support in the standard cuML communicator. Algorithms requiring point-to-point messaging will not work when this is disabled. |
```

Comment:
But they might work with the MPI_COMMS instead?

Reply:
The MPI comms is just for running automated MNMG gtests. Currently, it's not something most users are going to be building often, if at all. This is why it's been turned off by default. I've been being explicit about stating "standard cuML communicator" for this reason. If you prefer, I could be more explicit about this difference.

Reply:
A note like that I think would be a good idea, I could see a user/dev benefitting from having that note